samples/bookinfo/add_inject_annotation.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Why not change the files directly? Those files don't change that much, and, since we are a fork, it makes sense to change the files directly and get rid of those sed black magics...

Reply:
I added this script so we can then simply pull in the files from upstream and run these two scripts to make the necessary changes without having to think about merging.

Reply:
Those are trivial files so I'm confident we don't have to think about merges. git will do its job pretty easily :)

Reply:
Honestly, I feel it's too easy to miss a detail like this annotation. It happened to me today and left me wondering why injection stopped working all of a sudden. That's why having some kind of centralized "here's how we need to modify the upstream files to get a working version" file. 

I'd actually also add a {{Makefile}} to automate the whole process.